src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -62,5 +89,14 @@ public RobotMap() {
 			new VictorSP(Port.PWM.rightDriveA), new VictorSP(Port.PWM.rightDriveB));
 		Component.shifter = new SolenoidShifters(Port.Pneumatics.shifterUp, Port.Pneumatics.shifterDown);
 		Component.chassis = new TankDriveShifting("2018-Chassis", Component.leftWheel, Component.rightWheel, Component.shifter);
+		CANEncoder armEncoder = new CANEncoder(-1);// TODO: get the real id from electronics
```

Comment:
This ID is given through the CAN{} class where you put the port, so don't set this directly to -1. Instead, set it to that variable.